src/droplet.F

```diff
@@ -862,7 +862,8 @@ subroutine droplet_driver(dt,dbldt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,
       end if    ! if floor(mtime/drop_inject_elapse) .ne. floor((mtime+dt)/drop_inject_elapse) 
       !This is helpful to see in the output, but not absolutely necessary:
 #ifdef MPI
-      call mpi_allreduce(nparcelsLocalActive,np_tmp,1,mpi_int,mpi_sum,mpi_comm_world,ierr)
+      call mpi_allreduce(real(nparcelsLocalActive),np_tmp,1,mpi_real, &
```

Comment:
This seems like a problematic solution.  So MPI_INTEGER8 is not supported in the current MPI implementation?  Is there a way to use the C version of 64-bit integers?

Reply:
Revised as suggested.